ee/maintained-apps/README.md

```diff
@@ -66,3 +66,21 @@ These are command lines that will be run _before_ the generated uninstall script
 
 #### `post_uninstall_scripts`
 These are command lines that will be run _after_ the generated uninstall script is executed.
+
+### Testing
+
+Fleet tests every Fleet-maintained app. For new apps, start at step 1. For updates to existing apps, skip to step 4.
+
+1. When a pull request is opened in `inputs/`, the [#g-software Engineering Manager (EM)](https://fleetdm.com/handbook/company/product-groups#software-group) is automatically added as reviewer.
+2. The EM is responsible for making sure that the `name` for the new app matches the name that shows up in Fleet's software inventory. If the name doesn't match or if the name is not user-friendly, the EM will bring it to #g-software design review. This way, when the app is added to Fleet, the app will be matched with the app that comes back in software inventory.
+3. Then, the EM builds the app's `outputs/` on the same PR. At this time, @eashaw and a Product Designer are added to the PR. Eric adds the icon for [fleetdm.com/app-library](https://fleetdm.com/app-library).
+4. The Product Designer is responsible for testing the app:
+- If the app is a new app, add an icon for the app to the PR. App icons are located [here in the repository](https://github.com/fleetdm/fleet/tree/main/frontend/pages/SoftwarePage/components/icons). Run the [Docker publish GitHub action]() to add the tag to [Docker Hub](https://hub.docker.com/r/fleetdm/fleet/tags).
+- Run the following `fleetctl preview` command and test the app: Does the icon look right? Does the app install? Does the app uninstall? Can you open the app once it's installed?
+
+```
+fleetctl preview --tag=<Docker-tag> --license-key=eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJGbGVldCBEZXZpY2UgTWFuYWdlbWVudCBJbmMuIiwiZXhwIjoxNjQwOTk1MjAwLCJzdWIiOiJkZXZlbG9wbWVudCIsImRldmljZXMiOjEwMCwibm90ZSI6ImZvciBkZXZlbG9wbWVudCBvbmx5IiwidGllciI6ImJhc2ljIiwiaWF0IjoxNjIyNDI2NTg2fQ.WmZ0kG4seW3IrNvULCHUPBSfFdqj38A_eiXdV_DFunMHechjHbkwtfkf1J6JQJoDyqn8raXpgbdhafDwv3rmDw
```

Comment:
To get the latest apps we want to run `fleetctl preview` with this env variables:

```
FLEET_DEV_MAINTAINED_APPS_BASE_URL=https://raw.githubusercontent.com/fleetdm/fleet/refs/heads/fma-2505021402/ee/maintained-apps/outputs
```

@xpkoala do you know how to run fleetctl preview with [Fleet server environment variables](https://fleetdm.com/docs/configuration/fleet-server-configuration)?

Reply:
> 1. Edit `./tools/osquery/in-a-box/docker-compose.yml` by adding `FLEET_DEV_MAINTAINED_APPS_BASE_URL=https://raw.githubusercontent.com/fleetdm/fleet/refs/heads/fma-2505021402/ee/maintained-apps/outputs` after
line `FLEET_OSQUERY_HOST_IDENTIFIER: ${FLEET_OSQUERY_HOST_IDENTIFIER:-provided}` (in the two locations it's defined, because preview starts two fleet instances, one to connect to osquery and the other one for the webapp).

After making this edit, do we have to build a new `fleetctl` off of `main`? Or will the latest `fleetctl` "just work"?

Reply:
> Or will the latest fleetctl "just work"?

Latest `fleetctl` should work AFAICS.

PS: As long as the environment variable `FLEET_DEV_MAINTAINED_APPS_BASE_URL` is supported on the latest version of Fleet (`v4.67.2`).

Reply:
That env var has been there since 4.66.0 so we're good there.